src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -159,8 +159,13 @@ setTimeout(async function () {
       });
     }
 
+    let body = JSON.stringify({
+      "id": deltas[0].sequenceId,
+      "deltas": deltas
+    });
+
```

Comment:
Nitpick on syntax, hope that's OK:

```suggestion
    const body = JSON.stringify({
      id: deltas[0].sequenceId,
      deltas: deltas
    });
```

Reply:
Sure, thanks for the suggestion.